File: White or Black?.py
def square_color(file, rank):
    alpha = "abcdefgh"
    index=alpha.index(file)+1
    if (index%2==1 and rank%2==1) or (index%2==0 and rank%2==0):
        return "black"
    else:
        return "white"

File: White or Black?.js
function mineColor(file, rank) {
  const alpha = "abcdefgh"
  const index=alpha.indexOf(file)+1
  if ((index%2==1 && rank%2==1) || (index%2==0 && rank%2==0))return "black"
  else return "white"
}
